.github/workflows/ci.yml

```diff
@@ -52,6 +58,33 @@ jobs:
     - name: Checkout security
       uses: actions/checkout@v4
 
+    - name: Insert Admin Credential on Linux
```

Comment:
What is broken without this section?

Reply:
Based on the [log](https://github.com/opensearch-project/security/actions/runs/6399851879/job/17372609229?pr=3449#step:5:7183):
```
 - org.opensearch.security.InitializationIntegrationTests.testDefaultConfig
 - org.opensearch.security.InitializationIntegrationTests.testInvalidDefaultConfig
 - org.opensearch.security.TransportUserInjectorIntegTest.testSecurityUserInjection
 - org.opensearch.security.SlowIntegrationTests.testDelayInSecurityIndexInitialization
 ```
 For example, the test of  `InitializationIntegrationTests.testDefaultConfig` is using the admin admin credential to send out the request: https://github.com/opensearch-project/security/blob/93f79f881a718f35596dcd44e705b29b8c5c25c3/src/test/java/org/opensearch/security/InitializationIntegrationTests.java#L287